### PR TITLE
Improve CSS build VC-1651

### PR DESCRIPTION
**Pull request by say2me**

### CODE REVIEW

**Comment by say2me** [comment]
Merging nodes for mixins, attributes and styles gave nothing

Now system use saved information about CSS data of elements on initial load. It improves speed for 30% in average.

**Review by alperbuulut — APPROVED** [review]

File: public/editor/stores/assets/lib/cssBuilder.js [hunk 1]
```diff
@@ -283,18 +283,26 @@ export default class CssBuilder {
...
-            file.innerHTML = elementEditorCss[0].src
-          }
+          const styles = this.stylesManager.create()
+          styles.add(elementEditorCss)
+          this.addJob(styles.compile().then((result) => {
+            const file = this.window.document.getElementById(`vcv-css-editor-styles-${tag}`)
+            if (file) {
+              file.innerHTML = elementEditorCss[0].src
+            }
+          }))
```
Comment (konutis) [inline public/editor/stores/assets/lib/cssBuilder.js:305]: This shouldn't be changed. These are changes from me and Pavel. It removes compiling for editor.css files (it is not needed here).

File: public/editor/stores/assets/lib/cssBuilder.js [hunk 2]
```diff
@@ -355,27 +372,44 @@ export default class CssBuilder {
...
+          this.addedBaseStylesTagList.push(tag)
+          const styles = this.stylesManager.create()
+          styles.add(elementCssBase)
+          const css = styles.compile().then((result) => {
+            const style = this.window.document.getElementById(`vcv-base-css-styles-${tag}`)
+            if (style) {
+              style.innerHTML = elementCssBase[0].src
+            }
+          })
+          this.addJob(css)
```
Comment (konutis) [inline public/editor/stores/assets/lib/cssBuilder.js:397]: This shouldn't be changed. These are changes from me and Pavel. It removes compiling for styles.css files (it is not needed here).

**Review by pavelthq — CHANGES_REQUESTED** [review]
Please fix @konutis comments about wrong merge resolve

**Review by pavelthq — APPROVED** [review]

**Review by konutis — APPROVED** [review]

**Pull request merged**